pre_commit/clientlib/validate_config.py

```diff
@@ -52,8 +59,19 @@ def try_regex(repo, hook, value, field_name):
 
 
 def validate_config_extra(config):
+    local_repos_found = False
     for repo in config:
-        if 'sha' not in repo:
+        if is_local_hooks(repo):
+            if local_repos_found:
+                raise InvalidConfigError(
+                    'Multiple local hooks found'
+                )
```

Comment:
Yes, but I prefer to be strict at this early stage of config parsing, instead of allowing weird anomalies later.
E.g. if one needs a list of all the local hooks and assume there can be only one entry for them in the config.
Better avoid ambiguity, don't you think ?